var/spack/repos/builtin/packages/nvidia-cuda-samples/package.py

```diff
@@ -0,0 +1,60 @@
+# Copyright Spack Project Developers. See COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+import shutil
+
+from spack.package import *
+
+
+class NvidiaCudaSamples(CMakePackage, MakefilePackage, CudaPackage):
+    """A collection of NVIDIA curated sample applications showcasing features of CUDA"""
+
+    homepage = "https://github.com/NVIDIA/cuda-samples"
+    url = "https://github.com/NVIDIA/cuda-samples"
+    git = "https://github.com/NVIDIA/cuda-samples"
```

Comment:
Git URLs need to be complete. See the example at https://spack.readthedocs.io/en/latest/packaging_guide.html#git.

```suggestion
    git = "https://github.com/NVIDIA/cuda-samples.git"
```